clog/logger.go

```diff
@@ -23,9 +20,8 @@ var (
 )
 
 type clogger struct {
-	otel log.Logger
```

Comment:
can you please add a note about why we shouldn't use the otel logger here?  That'll help me remember the problem when i get around to adding interfaces.